packages/tasks/package.json

```diff
@@ -25,6 +25,7 @@
 		"format:check": "prettier --check .",
 		"prepublishOnly": "pnpm run inference-codegen && git diff --name-only --exit-code src && pnpm run build",
 		"build": "tsup src/index.ts --format cjs,esm --clean && tsc --emitDeclarationOnly --declaration",
+		"watch": "tsup src/index.ts --format cjs,esm --clean --watch",
```

Comment:
there's no need for the `tsc --declaration` part of the watch command?

Reply:
oh, yes, I thought it was only for type checking but it generates the declarations map too. Fixing it